nixos/tests/neo4j.nix

```diff
@@ -2,19 +2,25 @@ import ./make-test-python.nix {
   name = "neo4j";
 
   nodes = {
-    master =
+    server =
       { ... }:
 
       {
+	virtualisation.memorySize = 4096;
+	virtualisation.diskSize = 1024;
+
         services.neo4j.enable = true;
+        # require tls certs to be available
+        services.neo4j.https.enable = false;
```

Comment:
I wonder why those default to true.
Most services i know about always expose only http by default. https is usually taken care of by a reverse proxy.